python/oneflow/env.py

```diff
@@ -28,11 +28,19 @@
 
 
 def get_local_rank():
+    """Returns the local rank of current machine.
+    Local rank is not globally unique, it is only unique per process on a machine. 
+
+    Returns:
+        The the local rank of process on current machine.
+
+    """
     return oneflow._oneflow_internal.GetLocalRank()
 
 
 def get_rank():
     """Returns the rank of current process group.
+    Local rank is globally unique, range of which is from 0 to world_size. 
```

Comment:
```suggestion
    Global rank is globally unique, range of which is from 0 to world_size. 
```

这里的 rank 不是 local 的了吧？

Reply:
>这里的 rank 不是 local 的了吧？

已修改，没必要是```Global rank```，直接是Rank就行